packages/components/src/dropdown/styles.ts

```diff
@@ -36,3 +36,12 @@ export const DropdownContentWrapperDiv = styled.div< DropdownContentWrapperProps
 
 	${ padding };
 `;
+
+export const DropdownPointerEventsCapture = styled.div`
+	position: fixed;
+	inset: 0;
+	// Hardcoded fallback to (popover-z-index - 1)
+	z-index: 999999;
+	// TODO: remove following background color
+	background-color: rgba( 0, 0, 0, 0.4 );
```

Comment:
The backdrop is purposefully tinted to better showcase the solution proposed. This will be removed before merging, leaving the backdrop transparent